resources_pages/install_ds_stack_mac.md

```diff
@@ -518,15 +562,38 @@ This requires the `pyppeteer` package,
 which we can install as follows.
 
 ```bash
-conda install pyppeteer=0.2.2
+conda install pyppeteer
 pyppeteer-install
 ```
 
 Try this by going to `File -> Export notebook as... -> Export Notebook to WebPDF`.
 
+## Pandoc
+
+Install Pandoc for MacOs from the following link [Pandoc](https://pandoc.org/installing.html).
+
+Try in the command line
+
+```bash
+pandoc --version
+```
+You sould get this output:
+
+```bash
+pandoc 2.19
+Compiled with pandoc-types 1.22.2, texmath 0.12.5.2, skylighting 0.13,
+citeproc 0.8.0.1, ipynb 0.2, hslua 2.2.1
+Scripting engine: Lua 5.4
+User data directory: /Users/florenciadandrea/.local/share/pandoc
+Copyright (C) 2006-2022 John MacFarlane. Web:  https://pandoc.org
+This is free software; see the source for copying conditions. There is no
+warranty, not even for merchantability or fitness for a particular purpose.
+```
+
+
```

Comment:
I think the best solution for this is to avoid using a separate installation of pandoc inside and outside RStudio. I added some comments in https://github.com/UBC-MDS/UBC-MDS.github.io/issues/122#issuecomment-1216212482, and a fix for the setup script in #143

Reply:
Perfect, I agree that we can remove it

Reply:
It looks like this is still in the merged PR

Reply:
I deleted this in one of my commits to https://github.com/UBC-MDS/UBC-MDS.github.io/pull/147